.github/workflows/branch-merge.yml

```diff
@@ -0,0 +1,33 @@
+name: Auto-merge branches 
+
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  # It was decided to don't run the workflow automatically for now
+  #schedule:
+  #  - cron: '0 0 * * *'
+
+env:
+  REPO: https://cgeo:${{secrets.GITHUB_TOKEN}}@github.com/cgeo/cgeo.git
```

Comment:
I'm not sure, if it needs to run on behalf of the organization (`https://cgeo:`) or an individual user (e.g. `https://cgeo-ci-bot:`). Probable, we simply need to try out which one works...